src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -53,6 +82,15 @@
              :on-fail     #(log/error "Failed to pair with dApp" {:error %})
              :on-success  #(log/info "dApp paired successfully")}]]})))
 
+(rf/reg-event-fx
+ :wallet-connect/fetch-active-sessions
+ (fn [{:keys [db]}]
+   (let [web3-wallet (get db :wallet-connect/web3-wallet)]
+     {:fx [[:effects.wallet-connect/fetch-active-sessions
+            {:web3-wallet web3-wallet
+             :on-fail     #(log/error "Failed to get active sessions" {:error %})
```

Comment:
`on-fail` can be `on-error` for consistency: https://github.com/status-im/status-mobile/pull/20199#discussion_r1617772934